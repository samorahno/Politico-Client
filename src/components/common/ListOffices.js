import React, { Fragment } from 'react';

const ListOffices = ({ offices }) => {
    return (
        <div className="widget widget_table" style={{ overflow: 'auto' }}>
            <div className="title heado">Political Offices List</div>

            <div className="widget_stat widget_stat_table">
                <div style={{ overflow: 'auto' }}>
                    <table style={{ overflow: 'scroll' }}>
                        <thead>
                            <tr>
                                <th>S/N</th>
                                <th>Name</th>
                                <th>Type</th>

                            </tr>
                        </thead>


                        {offices && offices.map((office, index) => {
                            return (<Fragment key={office.id}>
                                <tbody id="officeContainer">
                                    <tr>
                                        <td>{index + 1}</td>
                                        <td>{office.name}</td>
                                        <td>{office.type}</td>
                                    </tr>
                                </tbody>
                            </Fragment>);
                        })}


                    </table>
                </div>
            </div>
        </div>
    );
};

export default ListOffices;
