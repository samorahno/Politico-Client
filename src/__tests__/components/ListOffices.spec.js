import React from 'react';
import { shallow } from 'enzyme';
import ListOffices from '../../components/common/ListOffices';

describe('List Offices Component', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<ListOffices />);
        expect(component).toMatchSnapshot();
    });
});
