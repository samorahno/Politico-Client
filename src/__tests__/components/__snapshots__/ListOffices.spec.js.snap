// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List Offices Component should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
